main.js

```diff
@@ -40,19 +54,20 @@ var logstartpoint = [-87, -320, -553, -181, -508, -835, -120, -386, -652];
 
 var breakpoint = [-27, 470, -30, 470, -50, -33, -33];
 
-var startpoint1 = [440, -30, 440, -30, 440, 440, 440];
+var startpoint1 = [game.width, -30, game.width, -30, game.width, game.width, game.width];
 var startpoint2 = [613, -206, 616, -206, 636, 473, 473];
 var startpoint3 = [759, -382, 792, -382, 832, 506, 506];
 
+// moved desty , destwidth and destheight elements from Obstacle objects in obstacleArray
+// to own arrays for easier manipulation
+var desty = [490, 450, 410, 370, 330, 250, 210, 170, 130, 120];
```

Comment:
I have a couple comments:
* Those heights could probably all be set to 25 making the array of different heights just a single variable.
* the widths could be changed into multiples of 30 to make it even more simple

Reply:
Like this, Christian?
var destwidth = 25; // then i would remove indexing from destwidth in Obstacles.
var desty= [490, 460, 400, 370, 340, 260, 200, 170, 130, 100];